models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+      - name: ind32
+        description: Indicator for indirect valuation
+      - name: ind33
+        description: Indicator for indirect valuation
+      - name: ind34
+        description: Indicator for indirect valuation
+      - name: ind35
+        description: Indicator for indirect valuation
+      - name: ind36
+        description: Indicator for indirect valuation
+      - name: ind37
+        description: Indicator for indirect valuation
+      - name: ind38
+        description: Indicator for indirect valuation
+      - name: ind39
+        description: Indicator for indirect valuation
+      - name: ind40
+        description: Indicator for indirect valuation
+      - name: itbld
+        description: Infotype screen control
+      - name: itxex
+        description: Text exists for infotype
+      - name: lga01
+        description: Wage type
+      - name: lga02
+        description: Wage type 
+      - name: lga03
+        description: Wage type 
+      - name: lga04
+        description: Wage type 
+      - name: lga05
+        description: Wage type 
+      - name: lga06
+        description: Wage type 
+      - name: lga07
+        description: Wage type 
+      - name: lga08
+        description: Wage type 
+      - name: lga09
+        description: Wage type 
+      - name: lga10
+        description: Wage type 
+      - name: lga11
+        description: Wage type 
+      - name: lga12
+        description: Wage type 
+      - name: lga13
+        description: Wage type 
+      - name: lga14
+        description: Wage type 
+      - name: lga15
+        description: Wage type 
+      - name: lga16
+        description: Wage type 
+      - name: lga17
+        description: Wage type 
+      - name: lga18
+        description: Wage type 
+      - name: lga19
+        description: Wage type 
+      - name: lga20
+        description: Wage type 
+      - name: lga21
+        description: Wage type 
+      - name: lga22
+        description: Wage type 
+      - name: lga23
+        description: Wage type 
+      - name: lga24
+        description: Wage type 
+      - name: lga25
+        description: Wage type 
+      - name: lga26
+        description: Wage type 
+      - name: lga27
+        description: Wage type 
+      - name: lga28
+        description: Wage type 
+      - name: lga29
+        description: Wage type 
+      - name: lga30
+        description: Wage type 
+      - name: lga31
+        description: Wage type 
+      - name: lga32
+        description: Wage type 
+      - name: lga33
+        description: Wage type 
+      - name: lga34
+        description: Wage type 
+      - name: lga35
+        description: Wage type 
+      - name: lga36
+        description: Wage type 
+      - name: lga37
+        description: Wage type 
+      - name: lga38
+        description: Wage type 
+      - name: lga39
+        description: Wage type 
+      - name: lga40
+        description: Wage type 
```

Comment:
Are these all intended to be the same description?

Reply:
Indeed <img width="728" alt="Screenshot 2023-07-06 at 4 20 48 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/638c0355-ef78-4bbf-9f2a-a919f79960e2">